### PR TITLE
Fix install VMware

### PR DESCRIPTION
**Pull request by Lennonka**
The VMware provider is installed by default in Satellite.
The navigation to verify the CR provider is installed has been fixed too.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_installing-vmware-plugin.adoc [hunk 1]
```diff
@@ -11,4 +11,6 @@ This allows you to manage and deploy hosts to VMware.
 ----
 # {foreman-installer} --enable-foreman-compute-vmware
 ----
-. Optional: In the {ProjectWebUI}, navigate to *Administer > About* and select the _compute resources_ tab to verify the installation of the VMware plugin.
+
+.Verification
+* In the {ProjectWebUI}, navigate to *Administer > About* and select the _Available Providers_ tab to verify the installation of the VMware plugin.
```
Comment (bangelic) [inline guides/common/modules/proc_installing-vmware-plugin.adoc:16]:
```suggestion
* In the {ProjectWebUI}, navigate to *Administer > About* and under *System Status*, select the _Available Providers_ tab to verify the installation of the VMware plugin.
```

**Review by bangelic — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_installing-vmware-plugin.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_installing-vmware-plugin.adoc:16]:
```suggestion
* In the {ProjectWebUI}, navigate to *Administer* > *About*.
* Under *System Status*, select the *Available Providers* tab to verify the installation of the VMware plugin.
```

Building upon Brian's suggestions, I think we should split these sentences.

**Review by maximiliankolb — COMMENTED** [review]
one open comment, rest LGTM

File: guides/common/assembly_provisioning-virtual-machines-vmware.adoc [hunk 2]
```diff
@@ -5,7 +5,9 @@
 
 include::modules/con_provisioning-virtual-machines-in-vmware.adoc[]
 
+ifndef::satellite[]
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/assembly_provisioning-virtual-machines-vmware.adoc:8]: CR providers are all enabled by default in Satellite, correct?

[the remark this answers is not in the document]
Comment (stejskalleos) [reply guides/common/assembly_provisioning-virtual-machines-vmware.adoc:8]: Aah ok, understood

**Comment by Lennonka** [comment]
I've updated some wordings. "The" provider is only one, compute resources can be multiple.

Pretty please for a final review :)

**Review by bangelic — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- 0ae938af3c..1aec858054  3.11 -> 3.11
- 60ea463de6..2dca1a43e5  3.10 -> 3.10
- b1d48c6309..fe5eb2a8eb  3.9 -> 3.9
- 19582ecff2..f2493a2127  3.8 -> 3.8
- 09a30ad6bf..5bb87bd4da  3.7 -> 3.7
- a307263224..648bac1ad5  3.6 -> 3.6
- 2c4259f1bd..4224dc1ecd  3.5 -> 3.5
- 554cc10e4a..0071ab6b86  3.4 -> 3.4
- 9ca4dc156e..a09b73955b  3.3 -> 3.3

**Pull request merged**